1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -21,22 +21,34 @@ instead!
 3. Complete the unit test functions and verify that all is working as expected.
 -----------------------------------------------------------------------------*/
 const cartForParty = {
-  // TODO complete this object
+  beer: 1.75,
+  chips: 2.59,
+  water: 1.1,
+  apple: 3.3,
+  banana: 2.3,
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
-  // TODO replace this comment with your code
+function calculateTotalPrice(object) {
+  let totalAmount = 0;
+  for (const value of Object.values(object)) {
```

Comment:
Correct however, we can simplify it a little bit more:
 ```suggestion
    Object.values(obj).forEach((price) => (totalPrice += price));
```